docs/connectors.md

```diff
@@ -18,12 +23,69 @@ Give the issue a `+1` if you really need that connector. The upvote will help th
 particular connector. If you find that an issue hasn't been created for your data store, please create a new issue in
 the Conduit repo.
 
-## Connectors
+## More about Connectors
+
+### Built-in connectors
+
+To help developers bootstrap pipelines much more quickly,
+Conduit ships with several built-in connectors by default.
+This includes the Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+If you are creating and distributing customized, pre-built Conduit binaries yourself,
+you may want to modify the list of built-in connectors, too.
+By modifying the list of built-in connectors,
+your compiled binary will include a set of pre-installed connectors specific to your end users' needs,
+and it can be run by others, as is, without them having to follow any additional installation instructions.
+
+If you want to create your own Conduit binary with a different set of built-in connectors,
+you can build-in an [existing standalone connector](#the-list).
+Alternatively, you can also create your own
+[using the Conduit connector template or the Conduit Connector SDK](https://conduit.io/docs/connectors/building/).
+
+Once you have chosen a connector to be built-in, you can:
+
+- Import the Github repository defining the connector
+into the [builtin registry](https://github.com/ConduitIO/conduit/blob/main/pkg/plugin/builtin/registry.go)
+and add a new key to the `DispenserFactory`:
+
+```go
```

Comment:
My suggestion would be to use `diff` for code highlighting and mark the added lines.
```diff
package builtin

import (
  // ...
+ myNewConnector "github.com/foo/conduit-connector-new"
)
```

Reply:
Oh I see, I didn't know about this code highlight feature before, this is neat!